qa/tasks/ceph.py

```diff
@@ -426,12 +426,20 @@ def cephfs_setup(ctx, config):
     # If there are any MDSs, then create a filesystem for them to use
     # Do this last because requires mon cluster to be up and running
     if mdss.remotes:
-        log.info('Setting up CephFS filesystem...')
+        log.info('Setting up CephFS filesystem(s)...')
+        cephfs_configs = config.get('cephfs', {'cephfs': {}})
+        set_allow_multifs = True if len(cephfs_configs) > 1 else False
+
+        for name, cephfs_config in cephfs_configs.items():
+            if cephfs_config is None:
+                cephfs_config = {}
+            if set_allow_multifs:
+                cephfs_config['set_allow_multifs'] = True
```

Comment:
Rather than set that as part of the "filesystem" config. Let's just call `fs.set_allow_multifs()` after creating the first one.